structure_tests/tests/cloudbuild.yaml.in

```diff
@@ -0,0 +1,5 @@
+steps:
+        - name: gcr.io/gcp-runtimes/structure-test-test:${TEST_TAG}
+          args: [
+          '--image', 'gcr.io/google-appengine/debian8:latest',
```

Comment:
So you want to run the structure tests... on the structure tests. Meta. It makes sense, I'll do that.